src/code.c

```diff
@@ -144,15 +144,22 @@ int blength;
   WriteComment("       R. Sander, Max-Planck Institute for Chemistry, Mainz, Germany");
   WriteComment("");
   WriteComment("%-20s : %s", "File", bufname );
-  strcpy( buf, (char*)ctime( &t ) ); 
+  strcpy( buf, (char*)ctime( &t ) );
   buf[ (int)strlen(buf) - 1 ] = 0;
-  WriteComment("%-20s : %s", "Time", buf );
-  WriteComment("%-20s : %s", "Working directory", getcwd(buf, MAX_PATHLEN) );
+//===========================================================================
```

Comment:
This was a modification I made for GEOS-Chem, but I can block this out with an `if (rootFileName = "gckpp")` block if needed.

Reply:
Although I like time stamps, I agree that they don't fit very well into the git philosophy. It's fine for me to remove these lines.

Minor comment: If you want to keep the lines as comments, I suggest to replace the 200 by MAX_PATHLEN. Just in case we want to reactivate it in the future and don't remember what the 200 means...